daily_report/PGConnection.py

```diff
@@ -0,0 +1,60 @@
+import psycopg2
+import pandas as pd
+import sqlalchemy
+import threading
```

Comment:
What does threading do?

Reply:
threading is for reading the sqlite table multiple times in quick succession. Parallelises the queries